app/presenters/fully_signed_in_modal_presenter.rb

```diff
@@ -1,22 +1,29 @@
 class FullySignedInModalPresenter
-  include ActionView::Helpers::TagHelper
   include ActionView::Helpers::TranslationHelper
 
-  def initialize(time_left_in_session)
-    @time_left_in_session = time_left_in_session
+  def initialize(expiration)
+    @expiration = expiration
   end
 
-  def message
+  def message(view_context)
     t(
       'notices.timeout_warning.signed_in.message_html',
-      time_left_in_session: content_tag(:span, time_left_in_session, id: 'countdown'),
+      time_left_in_session: view_context.render_to_string(
+        CountdownComponent.new(expiration: expiration, start_immediately: false),
+      ),
     )
   end
 
-  def sr_message
+  def sr_message(view_context)
```

Comment:
Passing in `view_context` here feels ugly, though we need to be able to have access to render the component markup. Alternatively, there might be an option to change it so that the translation call occurs in the view itself.

Reply:
Addressed in [a88ad93](https://github.com/18F/identity-idp/pull/6023/commits/a88ad9358333ea4a17bffa73d229fdd75a31e2a2).

Related discussion: https://github.com/18F/identity-idp/pull/6023#discussion_r817995755